website/content/en/docs/best-practices/multi-tenancy.md

```diff
@@ -0,0 +1,187 @@
+- **Should** provide the mechanisms for [Monitoring compute & memory resource usage][k8s-metrics]. Kubebuilder and SDK provide some [metrics][kb-metrics] 
+which can help you with. Note that cluster admins can use it to monitor and resize the Operator and its Operands.
+**NOTE**  If the Operator is integrated with OLM and the bundle has a `PodMonitor` or a `ServiceMonitor` the 
+complete `InstallPlan` will fail on a cluster, which does not have these CRD/the Prometheus operator installed. 
+In this case, you might want to ensure the dependency requirement with [OLM dependency][olm-dep] or make clear 
+its requirement for the Operator consumers. 
+- **Should** allow admins to customize the `requests/limits` amounts defined for the `Pod/Deployment`
+created by the Operator and not keep them fixed on the code.
+- **Should** document how your Operator consumer can customize/rightsize the resources requests and limits for the 
+Operator and Operands `Pod/Deployments` or let its consumers know that the solution could adopt a better approach where 
+the Operator automatically adjust the values to the environment rather than asking its consumers to amend them
+
+You might also consider leveraging the Vertical Pod Autoscaler to have the resources requested by the Operator 
+automatically adjusted to the cluster where it is deployed.[More info][k8s-vertical-pod-scaling]. You might also 
+look at allowing [horizontal pod autoscaling][k8s-autoscaling] for the other `Pod/Deployments` created by your project. 
+In this case, be aware that resource requests are also required for horizontal pod autoscaling to work.
+
+**NOTE:** If you are using OpenShift as Kubernetes distribution you might want to check the 
+doc [Automatically adjust pod resource levels with the vertical pod autoscaler
+][ocp-pod-scaling]. Caveat: If the VPA CRD is not available in the cluster where the operator 
+gets deployed the InstallPlan will fail. PLease, see how to work with [OLM dependency][olm-dep] 
+if your project integrating with.
+
+### But, what happens
+
+#### when the resource requests are not set?
+
+**If not set:** 
+- configurations made by the cluster administrators such as [ResourceQuota][k8s-resources-quotas] might not work with the usage of the LimitRanger admission controller to force defaults might not work.
+- the Operators consumers might face resource shortage on a node when resource usage later increases, for example, during a daily peak in request rate.
+- the Operators consumers might will be unable to install the Operator because does not have the minimal resources available.
+- the scheduler cannot make an informed placement choice when it picks the nodes the operator pods will be running on
+- when there is memory contention on the node the pod is likely to get either evicted or OOM killed
+- when there is CPU contention on the node the pod is likely to get starved of CPU cycles making the operator unresponsive
+
+#### when the resource limits are not set?
```

Comment:
Again, this is awkward as a heading. Consider the following:

```suggestion
#### Resource Limits

What happens when the resource limits are not set?
```
Then go into your **If not set:** stuff below.